app/resources/app-base.html.template

```diff
@@ -14,14 +14,22 @@
 {% block title %}{{block.super}}{% endblock title %}
 
 {% block logo %}
-  {% if env.repo and env.action and env.action != "start" %}
-    {% if env.rtl %}
-      <a href="{{env.start_url}}"><img class="title-bar-button" src="{{env.global_url}}/forward_arrow.png" alt="Back"></a>
+  {% if not env.amp %}
+    {% if env.repo and env.action and env.action != "start" %}
+      {% if env.rtl %}
+        <a href="{{env.start_url}}"><img class="title-bar-button" src="{{env.global_url}}/forward_arrow.png" alt="Back"></a>
+      {% else %}
+        <a href="{{env.start_url}}"><img class="title-bar-button" src="{{env.global_url}}/back_arrow.png" alt="Back"></a>
+      {% endif %}
     {% else %}
-      <a href="{{env.start_url}}"><img class="title-bar-button" src="{{env.global_url}}/back_arrow.png" alt="Back"></a>
+      <img class="title-bar-button" id="menu-button" src="{{env.global_url}}/menu.png" alt="Menu" onclick="showSideMenu()">
     {% endif %}
   {% else %}
-    <img class="title-bar-button" id="menu-button" src="{{env.global_url}}/menu.png" alt="Menu" onclick="showSideMenu()">
+  <div class="amp-menu">
+   <a href="{{env.start_url}}">
+    <amp-img class="amp-title-bar-button" width="24" height="24" src="{{env.global_url}}/menu.png" alt="Top">
```

Comment:
amp-img tag automatically resized auto image fill for images, but in terms of design that is very bad so I overwrite this style, I know this is a bit hacky way to solve the issue.

Reply:
You mean that the image itself will be in 24x24 but it will have 20px left+right margin by specifying these? Can't you achieve the same by specifying "margin" or "padding" CSS attributes?

Reply:
Sorry, I did it in another css class, so I removed this overwriting to 64px part.